backoffice/backoffice-domain-service/ballerina/Dependencies.toml

```diff
@@ -379,23 +382,43 @@ modules = [
 	{org = "ballerinax", packageName = "postgresql", moduleName = "postgresql"}
 ]
 
+[[package]]
+org = "wso2"
+name = "apk_common_lib"
```

Comment:
let's add these to the dependencies toml template file as well

Reply:
Fix https://github.com/wso2/apk/pull/719/commits/083489a42f579d12f6a5e7214e371301ed10a2d3